system_strings/sysmes_text.en

```diff
@@ -1230,31 +1230,31 @@ I can't break my promise with^Arcueid. I have to somehow slip^behind their backs
 Well, isn't it because you want^revenge on me for killing you...
-To be honest, I'd like to ask^that myself. Because you're a^vampire... or something?
+To be honest, I'd like to know^that myself. Because you're a^vampire... or something?
 First, let's leave the^mansion. There's nowhere^to enjoy ourselves here.
-In order to entertain Arcueid,^we head to the direction of the^station... Wait, what is this^princess doing?
-A movie is standard fare.^Don't call it boring or something^like that. It is exactly because^it's standard that it's the^tried-and-true method.
-I wonder if I'm even sane.^But there's no doubt about it,^I must fulfill my overwhelming^desire.
+In order to entertain Arcueid,^we head to the direction of the^station... Wait, what's Her ^Majesty up to now?
+Going to the movies is a bit standard,^but don't go calling it boring.^It's the best choice; it's the standard^for a reason.
+I wonder if I'm even sane.^But this is the right choice, there's^no doubt about it,^I must fulfill my overwhelming^desire.
 Fortunately, today's weather^is great. I decide to bring^her along to the park.
-Talking about her. Talking^about food. Talking about^the enemy.
-After being pestered by Arcueid,^we ended up at my school. What^on earth does she plan to do...
+Talking about her.^Talking about food.^And... talking about the enemy.
+After being pestered by Arcueid,^we ended up at my school. What^on earth is she planning to do...
 Talking about 'what-ifs'.
 I arrived at the park, together^with Arcueid. It's too early to^begin our search, so we wait as^time passes by.
-'Test it out', she said. Her^breath approaches my nape. In^the face of this inexplicable^fear, I――
-Even though it was supposed to^be a joke. Arcueid apologizes^and leaps away.
-The Holy Church's Presbyter^speaks. About her 'enemy's'^true form, and his purpose.
+Test it out', she said. Her^breath approaches my nape. In^the face of this inexplicable^fear, I――
+Even though it was supposed to^be a joke, Arcueid apologizes^and leaps away.
+The Presbyter from the Holy Church^speaks about her enemy's^true form, and his purpose.
 I can't afford to waste any more^time. I rush into the nighttime^city to chase after Arcueid.
 I can't find Arcueid. If she was^here, she'd be――
-I'll go to her room. She might^have given up searching for the^Dead for tonight, and gone back^to her room.
-I don't think she went back^to her room. If she was to go^somewhere, it'd be somewhere^no one would go to―――
-I remove my glasses and head^into the nighttime city. She's^the only one whose death I can't^see. I'll believe that'll help^me find her.
+I'll go to her room. She might^have given up on searching for ^Dead tonight and gone back^to her room.
+I don't think she went back^to her room. If she was to go^somewhere, it'd be a place^no one would go to―――
+I remove my glasses and head^into the night. She's^the only one whose death I can't^see. I'll believe that'll help^me find her.
```

Comment:
`I'll believe` is a bit weird, I think just `I believe`? Maybe I should port this file to the includes-JP format

Reply:
just a typo. should be "I believe"